src/inputType/PanInput.ts

```diff
@@ -265,15 +309,25 @@ export class PanInput implements IInputType {
 				useDirection(DIRECTION_VERTICAL, this._direction, userDirection),
 			]);
 		const prevent = offset.some(v => v !== 0);
+
 		if (prevent) {
-			event.srcEvent.preventDefault();
-			event.srcEvent.stopPropagation();
+			const srcEvent = event.srcEvent;
+
+			if (srcEvent.cancelable !== false) {
```

Comment:
I think that if statement unnecessary because preventDefault works only when `srcEvent.cancelable` is true.

Reply:
@mixed 

Yes. But there are browsers without cancelable.

Reply:
@daybrush 
OK. If so, `srcEvent.cancelable === true` seems to be more readable.

Reply:
@mixed 

If there is no cancelable, preventDefault is possible. It is possible except when it is false.